librz/core/linux_heap_glibc.c

```diff
@@ -849,11 +856,17 @@ static bool GH(parse_tcache_from_addr)(RzCore *core, GHT tls_addr, GHT tid) {
 		}
 
 		rz_io_nread_at(core->io, tls_addr, tcache_addr, sizeof(GHT));
-		GHT tcache_guess = GH(read_val)(core, tcache_addr, false);
+		const GHT tcache_guess = GH(read_val)(core, tcache_addr, false);
 		if (tcache_guess < map->addr || tcache_guess > map->addr_end) {
 			continue;
 		}
 
+#if __aarch64__
```

Comment:
In the future, parsing should not depend on the architecture Rizin was built on.